src/buckets/components/createBucketForm/MeasurementSchemaSection.test.tsx

```diff
@@ -264,3 +266,60 @@ describe('test schema  update validity function', () => {
     doTest(null, true)
   })
 })
+
+describe('test csv conversion function (object->csv file)', () => {
+  it('should create a valid csv file', () => {
```

Comment:
i'd like to see a couple of tests that pass in bad data to make sure the method responds the way you expect

Reply:
the code  does the validating for bad data *after* it is converted to an object.  there are lots of unit tests for that.
if the csv is bad (it gets past the checks in the ui), the backend catches that.

Reply:
i spoke with Kristina about this, she said that due to the extensive unit testing, just need happy-path e2e tests

Reply:
although I just did add one more unit test for checking that the column headers are correct.  the rest is up to the back end/tests that check the content after parsing which are already there.

Reply:
@hoorayimhelping  done in a follow-up pr: https://github.com/influxdata/ui/pull/3229